core/src/main/java/org/geysermc/geyser/translator/protocol/java/level/JavaLevelEventTranslator.java

```diff
@@ -303,22 +338,23 @@ public void translate(GeyserSession session, ClientboundLevelEventPacket packet)
                 if (eventData.getCharge() > 0) {
                     levelEventPacket.setType(org.cloudburstmc.protocol.bedrock.data.LevelEvent.SCULK_CHARGE);
                     levelEventPacket.setTag(
-                            NbtMap.builder()
-                                    .putInt("x", packet.getPosition().getX())
-                                    .putInt("y", packet.getPosition().getY())
-                                    .putInt("z", packet.getPosition().getZ())
-                                    .putShort("charge", (short) eventData.getCharge())
-                                    .putShort("facing", encodeFacing(eventData.getBlockFaces())) // TODO check if this is actually correct
-                                    .build()
+                        NbtMap.builder()
+                            .putInt("x", packet.getPosition().getX())
+                            .putInt("y", packet.getPosition().getY())
+                            .putInt("z", packet.getPosition().getZ())
+                            .putShort("charge", (short) eventData.getCharge())
+                            .putShort("facing", encodeFacing(eventData.getBlockFaces())) // TODO check if this is actually correct
+                            .build()
                     );
                 } else {
                     levelEventPacket.setType(org.cloudburstmc.protocol.bedrock.data.LevelEvent.SCULK_CHARGE_POP);
+                    levelEventPacket.setTag("");
```

Comment:
???

Reply:
when i was testing stuff i probably accidentally added this line, my bad ill remove it soon

Reply:
Fixed in latest commit.